### ISSUE TITLE
Disable API requests for airports not served by backend

### ISSUE THREAD

**Issue by LeoKle** [vACDM/vacdm-plugin#34]
Using the endpoint /airports add a check if the backend is supporting the airport and only send requests for supported airports

**Comment by LeoKle** [comment]
completed in branch [fix/only-send-requests-for-supported-airports](https://github.com/vACDM/vACDM-Plugin/tree/fix/only-send-requests-for-supported-airports)

**Comment by LeoKle** [comment]
merged into dev, waiting on version 2 to deploy
